docs/source/_toctree.yml

```
@@ -41,23 +57,64 @@
       title: Inference Tutorials
     title: Tutorials
   - sections:
-    - local: guides/setup_aws_instance
-      title: Get Started on Amazon EC2
-    - local: guides/sagemaker
-      title: Training and Deployment using Amazon Sagemaker
-    - local: guides/cache_system
-      title: Neuron model cache
-    - local: guides/fine_tune
-      title: Fine-tune Transformers with AWS Trainium
-    - local: guides/distributed_training
-      title: Distributed Training
-    - local: guides/export_model
-      title: Export a model to Inferentia
-    - local: guides/pipelines
-      title: Inference pipelines with AWS Neuron
-    - local: guides/neuronx_tgi
-      title: NeuronX Text-generation-inference for AWS inferentia2
-    title: How-To Guides
+    - isExpanded: false
+      sections:
+      - isExpanded: false 
+        sections: 
+        - local: model_doc/transformers/bert
+          title: BERT
+        title: TEXT MODELS
+      - isExpanded: false
+        sections: 
+        - local: model_doc/transformers/yolos
+          title: YOLOS
+        title: VISION MODELS
+      - isExpanded: false 
+        sections: 
+        - local: model_doc/transformers/whisper
+          title: Whisper
+        title: AUDIO MODELS
+      - isExpanded: false
+        sections: 
+        - local: model_doc/transformers/clip
+          title: CLIP
+        title: MULTIMODAL MODELS
+      title: Transformers Models
+    - isExpanded: false
+      sections:
+      - local: model_doc/sentence_transformers/overview
+        title: Overview
+      title: SENTENCE TRANSFORMERS
+    - isExpanded: false
+      sections:
+      - local: model_doc/diffusers/controlnet
+        title: ControlNet
+      - local: model_doc/diffusers/pix2pix
+        title: InstructPix2Pix
+      - local: model_doc/diffusers/lcm
+        title: Latent Consistency Models
+      - local: model_doc/diffusers/pixart_alpha
+        title: PixArt-α
+      - local: model_doc/diffusers/pixart_sigma
+        title: PixArt-Σ
+      - isExpanded: false
+        sections: 
+        - local: model_doc/diffusers/stable_diffusion
+          title: Stable Diffusion
+        - local: model_doc/diffusers/stable_diffusion_xl
+          title: Stable Diffusion XL
+        - local: model_doc/diffusers/sdxl_turbo
+          title: SDXL Turbo
+        title: STABLE DIFFUSION
```

I think we could keep it in the same section? 

```suggestion
      - local: model_doc/diffusers/stable_diffusion
        title: Stable Diffusion
      - local: model_doc/diffusers/stable_diffusion_xl
        title: Stable Diffusion XL
      - local: model_doc/diffusers/sdxl_turbo
        title: SDXL Turbo
```

---

Yeah, it's not necessary to make another level, will do that in the next PR.